Tests/SentryTests/Integrations/SentryPerformanceTrackerTests.swift

```diff
@@ -0,0 +1,112 @@
+import XCTest
+
+class SentryPerformanceTrackerTests: XCTestCase {
+    
+    private static let dsnAsString = TestConstants.dsnAsString(username: "SentrySDKTests")
+    
+    private class Fixture {
+
+        let someTransaction = "Some Transaction"
+        let someOperation = "Some Operation"
+        
+        func getSut(initSDK: Bool = false) -> SentryPerformanceTracker {
+            if initSDK {
```

Comment:
The tracker sets the scope with a transaction, but only if the scope has no transaction already.
I have a test for this behavior.

But maybe we should go a step further and remove any SentrySDK reference from SentryPerformanceTracker and create a protocol to use as a delegate reducing coupling

Reply:
Then I would suggest to use `SentrySDK.setCurrentHub()` in combination with a `TestHub`. Then in `tearDown` you can set the current hub to nil. I just realized that `SentrySDK.setCurrentHub()` doesn't allow you to pass in `nil`. I'm going to change that.

Reply:
I added this suggestion except for the tearDown because as you mentioned is not possible to set the current hub to nil.

Reply:
But I think now we can remove `SentrySDK.start` because we set the hub below?

Reply:
I opened up a PR https://github.com/getsentry/sentry-cocoa/pull/1153

Reply:
It's not possible to ditch SentrySDK.start completely because we cannot set the Hub in SentryPerformanceTracker.
That's one refactoring I intend to do to reduce coupling.

Reply:
Yes, we can, I changed it to be able to merge it. Nevermind 😁